requirements-dev.txt

```diff
@@ -59,3 +59,4 @@ nvidia-ml-py
 huggingface_hub
 pyamg>=5.0.0
 git+https://github.com/Project-MONAI/GenerativeModels.git@7428fce193771e9564f29b91d29e523dd1b6b4cd
```

Comment:
@KumoLiu please remove this GenerativeModels dependency after refactoring MAISI.

Thanks.

Reply:
Sure, will remove it in maisi-related prs.